lib/api-client.js

```diff
@@ -5,6 +5,10 @@ var apiClient = new JSONAPIClient(config.host + '/api', {
   'Content-Type': 'application/json',
   'Accept': 'application/vnd.api+json; version=1',
 }, {
+  params: {
+    http_cache: true
```

Comment:
right - this is great for testing but can this be configured via URL params with a default fallback, ala [here](https://github.com/zooniverse/panoptes-javascript-client/blob/666306102e65795f33f2e235157610bad43adb8b/lib/config.js#L37)?